src/test/java/be/seeseemelk/mockbukkit/entity/EnderPearlMockTest.java

```diff
@@ -0,0 +1,68 @@
+package be.seeseemelk.mockbukkit.entity;
+
+import be.seeseemelk.mockbukkit.MockBukkitExtension;
+import be.seeseemelk.mockbukkit.MockBukkitInject;
+import be.seeseemelk.mockbukkit.ServerMock;
+import org.bukkit.Material;
+import org.bukkit.entity.EnderPearl;
+import org.bukkit.entity.EntityType;
+import org.bukkit.inventory.ItemStack;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+import java.util.UUID;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertNotSame;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+@ExtendWith(MockBukkitExtension.class)
+class EnderPearlMockTest
+{
+
+	@MockBukkitInject
+	ServerMock server;
+	EnderPearl pearl;
+
+	@BeforeEach
+	void setUp()
+	{
+		pearl = new EnderPearlMock(server, UUID.randomUUID());
+	}
+
+	@Test
+	void getItem_DefaultIsEnderPearl()
+	{
+		assertEquals(Material.ENDER_PEARL, pearl.getItem().getType());
+	}
+
+	@Test
+	void testSetItem()
+	{
+		ItemStack item = new ItemStack(Material.APPLE);
```

Comment:
```suggestion
		ItemStack item = new ItemStack(Material.APPLE, 64);
```
This is what I meant you should do to test amount of items properly.  ( btw ,if you run this test now, it will most definitely fail). Again, sorry for being a nitpick